src/main/java/org/openpnp/spi/base/AbstractPnpJobProcessor.java

```diff
@@ -36,7 +49,12 @@ public static void discard(Nozzle nozzle) throws Exception {
                 Configuration.get().getMachine().getDiscardLocation());
         // discard the part
         nozzle.place();
+		// repeat the air puff to clean filters over discard location
```

Comment:
It cannot handle by the driver as it is allowed only on discard location, it is for machines with
air puff  for 40ms as example. If machine don't have it, there is only a M9 statement
or similar, and any repeated call don't actuate any solenoid in this case after the first place.
If the nozzle hold one component, the solenoid is deactivated, otherwise it remains deactivated.
I can remove it.